### PR TITLE
chore: change default branch in pipeline manifest

### LINKED ISSUE #1481: pipeline: change default branch
In the pipeline manifest, we should change the default branch tracked by the pipeline to `main` to reflect GitHub's defaults.

See: 
- https://github.com/github/renaming
- https://tools.ietf.org/id/draft-knodel-terminology-00.html

### PR DESCRIPTION
**Pull request by SoManyHs**
Fixes #1481

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by SoManyHs** [comment]
Previous:
```
# This YAML file defines the relationship and deployment ordering of your environments.

# The name of the pipeline
name: pipeline-voting-app-SoManyHs-results

# The version of the schema used in this template
version: 1

# This section defines the source artifacts.
source:
  # The name of the provider that is used to store the source artifacts.
  provider: GitHub
  # Additional properties that further specifies the exact location
  # the artifacts should be sourced from. For example, the GitHub provider
  # has the following properties: repository, branch.
  properties:
    access_token_secret: github-token-voting-app-results
    branch: master
    repository: https://github.com/SoManyHs/results

# The deployment section defines the order the pipeline will deploy
# to your environments.
stages:
    - # The name of the environment to deploy to.
      name: test
      # Optional: flag for manual approval action before deployment.
      # requires_approval: true
      # Optional: use test commands to validate this stage of your build.
      # test_commands: [echo 'running tests', make test]

    - # The name of the environment to deploy to.
      name: prod
      # Optional: flag for manual approval action before deployment.
      # requires_approval: true
      # Optional: use test commands to validate this stage of your build.
      # test_commands: [echo 'running tests', make test]

```

Now:

```
# This YAML file defines the relationship and deployment ordering of your environments.

# The name of the pipeline
name: pipeline-voting-app-SoManyHs-results

# The version of the schema used in this template
version: 1

# This section defines the source artifacts.
source:
  # The name of the provider that is used to store the source artifacts.
  provider: GitHub
  # Additional properties that further specifies the exact location
  # the artifacts should be sourced from. For example, the GitHub provider
  # has the following properties: repository, branch.
  properties:
    access_token_secret: github-token-voting-app-results
    branch: main
    repository: https://github.com/SoManyHs/results

# The deployment section defines the order the pipeline will deploy
# to your environments.
stages:
    - # The name of the environment to deploy to.
      name: test
      # Optional: flag for manual approval action before deployment.
      # requires_approval: true
      # Optional: use test commands to validate this stage of your build.
      # test_commands: [echo 'running tests', make test]

    - # The name of the environment to deploy to.
      name: prod
      # Optional: flag for manual approval action before deployment.
      # requires_approval: true
      # Optional: use test commands to validate this stage of your build.
      # test_commands: [echo 'running tests', make test]

```

**Review by huanjani — APPROVED** [review]
:tada:

File: internal/pkg/cli/pipeline_init.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ Please enter full repository URL, e.g. "https://github.com/myCompany/myRepo", or
 const (
 	buildspecTemplatePath = "cicd/buildspec.yml"
 	githubURL             = "github.com"
-	masterBranch          = "master"
+	defaultBranch         = "main"
```
Comment (efekarakus) [inline internal/pkg/cli/pipeline_init.go:47]: nice rename!

**Review by efekarakus — APPROVED** [review]
Yay thank you!

**Pull request merged**